clients/ui/frontend/src/app/api/k8s.ts

```diff
@@ -1,10 +1,17 @@
-import { APIOptions } from '~/types';
+import { APIOptions } from '~/app/api/types';
 import { handleRestFailures } from '~/app/api/errorUtils';
-import { restGET } from '~/app/api/apiUtils';
+import { isModelRegistryResponse, restGET } from '~/app/api/apiUtils';
 import { ModelRegistry } from '~/app/types';
 import { BFF_API_VERSION } from '~/app/const';
 
-export const getModelRegistries =
+export const getListModelRegistries =
   (hostPath: string) =>
-  (opts: APIOptions): Promise<ModelRegistry> =>
-    handleRestFailures(restGET(hostPath, `/api/${BFF_API_VERSION}/model_registry`, {}, opts));
+  (opts: APIOptions): Promise<ModelRegistry[]> =>
+    handleRestFailures(restGET(hostPath, `/api/${BFF_API_VERSION}/model_registry`, {}, opts)).then(
```

Comment:
Just added here the conversion to make it easier to process in the rest of the dashboard.